app/(settings)/shareCircle.js

```diff
@@ -112,6 +150,76 @@ import QRCode from 'react-qr-code';
 					press={() => shareCircle()}
 				/>
 			</StyledView>
+
+			<StyledModal
+				className='w-[80%] self-center'
+				isVisible={isModalVisible1}
+				onBackdropPress={toggleModal1}
+			>
+				<StyledView className='bg-offblack border-[5px] border-purple rounded-2xl h-[55%] px-2 py-3'>
+					<StyledView className='items-center'>
+						<StyledText className='top-[6%] text-3xl text-offwhite'>
+							How to share circles:
+						</StyledText>		
+					</StyledView>
+					<StyledView className='flex-1 mx-2 gap-y-8'>
+						<StyledView className='flex-row items-center pt-6'>
+							<StyledIcon 
+								className=''
+								name='qr-code-outline' 
+								size={30} 
+								color='#FFFBFC'									
+							/>
+							<StyledText className='ml-3 text-2xl text-offwhite'>
+								Scan this QR code
+							</StyledText>
+						</StyledView>
+						<StyledView className='flex-row items-center'>
+							<StyledIcon 
+								className=''
+								name='text' 
+								size={30} 
+								color='#FFFBFC'									
+							/>
+							<StyledText className='ml-3 text-2xl text-offwhite'>
+								Share this code
+							</StyledText>
+						</StyledView>						
+						<StyledView className='flex-row items-center'>
+							<StyledIcon 
+								className=''
+								name='share-outline' 
+								size={30} 
+								color='#FFFBFC'									
+							/>
+							<StyledText className='ml-3 text-2xl text-offwhite'>
+								Share on other apps
+							</StyledText>
+						</StyledView>						
+						<StyledView className='flex-row items-center'>
+							<StyledIcon 
+								className=''
+								name='shield' 
+								size={30} 
+								color='#FFFBFC'									
+							/>
+							<StyledText className='ml-3 text-2xl text-offwhite'>
+								Share the private code
+							</StyledText>
+							<StyledText className='absolute top-10 left-10 text-l text-offwhite'>
+								Only share with trusted people!
+							</StyledText>
+						</StyledView>					
+					</StyledView>
+					<StyledView className='items-center'>
```

Comment:
This button is really close to the bottom of the modal. It should be equal to the spacing from the top